app/src/renderer/components/staking/LiValidator.vue

```diff
@@ -98,10 +105,10 @@ export default {
       return `n/a`
     },
     yourVotes() {
-      return this.committedDelegations[this.validator.id]
+      return this.committedDelegations[this.validator.operator_address]
         ? calculateTokens(
           this.validator,
-          this.committedDelegations[this.validator.id]
+          this.committedDelegations[this.validator.operator_address]
         )
         : BigNumber(0)
```

Comment:
Why BigNumber to represent a 0? is not part of this PR, just curious

Reply:
`calculateTokens` returns `BigNumber`. this way the return type stays predictable.

Reply:
nice, got it thanks!